### DISCUSSION TITLE
any plans for supporting screen readers?

### DISCUSSION THREAD

**Discussion by king-dahmanus** [coqui-ai/TTS#1635] [Ideas]
hello there. Are there any plans to: First, speed up the models so they generate and synthesize speech in less than 20ms with a cpu, then making a synthdriver for the opensource, free and extensible nvda screen reader found at https://github.com/nvaccess/nvda so blind people can use any models available to read their texts? Thanks

**Comment by erogol** [comment]
We don't have any plans for now.
